<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_Ahpha">
     <attribute name="title">
      <string>Alpha</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_a1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_phy"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>20</y>
        <width>31</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_a1">
         <property name="text">
          <string>a1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b1">
         <property name="text">
          <string>b1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a2">
         <property name="text">
          <string>a2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b2">
         <property name="text">
          <string>b2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a3">
         <property name="text">
          <string>a3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b3">
         <property name="text">
          <string>b3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a4">
         <property name="text">
          <string>a4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b4">
         <property name="text">
          <string>b4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_phy">
         <property name="text">
          <string>phy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Beta">
     <attribute name="title">
      <string>Beta</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>26</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_a1_2">
         <property name="text">
          <string>a1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b1_2">
         <property name="text">
          <string>b1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a2_2">
         <property name="text">
          <string>a2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b2_2">
         <property name="text">
          <string>b2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a3_2">
         <property name="text">
          <string>a3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b3_2">
         <property name="text">
          <string>b3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a4_2">
         <property name="text">
          <string>a4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b4_2">
         <property name="text">
          <string>b4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_phy_2">
         <property name="text">
          <string>phy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_a1_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b1_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a2_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b2_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a3_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b3_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a4_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b4_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_phy_2"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Gamma">
     <attribute name="title">
      <string>Gamma</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>26</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_a1_3">
         <property name="text">
          <string>a1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b1_3">
         <property name="text">
          <string>b1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a2_3">
         <property name="text">
          <string>a2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b2_3">
         <property name="text">
          <string>b2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a3_3">
         <property name="text">
          <string>a3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b3_3">
         <property name="text">
          <string>b3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_a4_3">
         <property name="text">
          <string>a4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_b4_3">
         <property name="text">
          <string>b4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_phy_3">
         <property name="text">
          <string>phy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="lineEdit_a1_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b1_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a2_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b2_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a3_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b3_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a4_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b4_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_phy_3"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_para_Set">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>曲线参数设置</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QWidget" name="curvePreview" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>500</width>
      <height>450</height>
     </rect>
    </property>
    <widget class="QChartView" name="chartView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>450</width>
       <height>350</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_linepreview">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>曲线预览</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Btn_control_Window">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>控制界面</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>206</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_Alpha">
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Beta">
       <property name="text">
        <string>Beta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Gamma">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
